src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -56,6 +58,12 @@ const CooperationDetails = () => {
     onResponseError: responseError
   })
 
+  useEffect(() => {
+    response.sections && response.sections.length
+      ? setEditMode(true)
+      : setEditMode(false)
+  }, [response.sections])
```

Comment:
Do we really need `useEffect` here? If we use following:

`const editMode = response.sections && response.sections.length`

Please try it

Reply:
Tried your solution but unfortunately, it didn't work